### DISCUSSION TITLE
A way to change admin password (not only a reset)

### DISCUSSION THREAD

**Discussion by RealOrangeOne** [plausible/analytics#230] [Anything Else]
## Prerequisites 
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report

**Describe the bug**

As a user, I cannot change my password

**Expected behavior**

I can change my password.

**Screenshots (If applicable)**

![image](https://user-images.githubusercontent.com/6527489/87811261-45157c00-c856-11ea-807d-13824bc43293.png)

**Environment (If applicable):**
 - OS: Linux (Docker)

**Comment by tckb** [comment]
At the moment, I have not looked into changing the password. The could be added as the next feature (this not a bug it was  never added) but this might have implication on the cloud version. @ukutaht  / @metmarkosaric  is this supported in the cloud offering?

**Comment by metmarkosaric** [comment]
This is the way we recommend cloud users to change passwords at this stage: https://docs.plausible.io/reset-password

**Comment by tckb** [comment]
@RealOrangeOne  If you have configured smtp, you can reset your admin password  at `<base_url>/password/request-reset`  until there's a better alternative, you can try this

**Comment by RealOrangeOne** [comment]
Using the password reset flow is an idea, never thought of that, thanks!

I think I meant this issue more as a feature request than a bug anyway!
